src/data-display/dynamic/dynamic-layout/templates/table/index.vue

```diff
@@ -31,6 +31,7 @@
                          @select="onSelect"
                          @export="onExport"
                          @click-settings="$emit('click-settings')"
+                         @rowLeftClick="onClickRow"
```

Comment:
Does DS write an event handler with the 'on' prefix?

Reply:
I don't know why exactly, but it only works with `on` prefix.